# Indeed: Syntect Server Readiness Error <!-- Ticket Title  Hint: include keywords to make it searchable -->
 
- Zendesk Link: [#4740](https://sourcegraph.zendesk.com/agent/tickets/4740)
- Application engineer: Alex Jean-Baptiste
- Customer: Indeed <!-- Redact if this contains personally identifying information -->
- Date: Nov 24

<!-- Data populated from integration, speak to Ben Gordon or Michael Bali if not working -->
<!-- During Internal team trial, fill missing data manually (we are waiting for all data to sync) -->
 
## Technical Environment
- Version: 3.33.2â€‹
- Deployment: kubernetes
- External Services: GITLAB
- Auth Providers: builtin,saml
 
 
## Links
<!-- Data for application engineer manual entry -->
- Slack Links: https://sourcegraph.slack.com/archives/CSKMGUJ58/p1637767334055900
- GitHub Issue Link: N/A
- Doc Update Link: N/A
 
## Summary
### Customer Question
The customer expressed a Readiness probe failed: dial TCP ${IP Address}:i/o timeout and Readiness probe failed: dial TCP ${IP Address}: connect: connection refused  errors from their syntect-server pod.

### Application Engineer Answer
The customer was advised to double the value of the initialDelaySeconds value in their syntect-server.Deployment.yamlfile. The customer never confirmed if this fixed their error.
 
### Relevant Docs Pages Used/Created
https://stackoverflow.com/questions/48540929/kubernetes-readiness-probe-failed-error

<!-- Once complete, upload a copy to https://github.com/sourcegraph/support-tools-internal/tree/main/resolved-tickets as a .md file -->
<!-- Name the file 4740.md -->
